lib/constants.ts

```diff
@@ -1,4 +1,5 @@
 export const CLOUD_TEMP_DIR_NAME = ".cloud";
+export const CODESIGN_TEMP_DIR_NAME = ".codesign";
```

Comment:
This does not seem like temp dir name, i.e. we download the certificate and provision there, it is inside the project and there's nothing temporarily in it.
Also why the name start with `.` - do we want to download the provision and certificate and leave the user wondering where are they?

Reply:
I wasn't sure - do we make them wonder where are the binaries? I supposed there should be some additional work for saving/using these. Wasn't sure. Anyway will remove the hidden name and temp identifier.